bindings/azure/storagequeues/storagequeues.go

```diff
@@ -177,9 +183,22 @@ func (d *AzureQueueHelper) Read(ctx context.Context, consumer *consumer) error {
 		}
 	}
 
+	metadata := make(map[string]string)
```

Comment:
Since we know that we're adding 6 keys (at most), let's pre-allocate with an initial capacity

```suggestion
	metadata := make(map[string]string, 6)
```

Reply:
Done